nodeServer/ts/helloWorld.ts

```diff
@@ -57,7 +109,7 @@ class HelloWorldDataservice{
 
 exports.helloWorldRouter = function(context): Router {
   return new Promise(function(resolve, reject) {
-    let dataservice = new HelloWorldDataservice(context);
+    let dataservice = new HelloWorldDataservice(context, process);
```

Comment:
Although the process object is global, in TS if we do not declare it explicitly, conditionals above like "process.clusterManager" will throw compilation errors